### PR TITLE
[CIVP-11093] DOC example civis.ml notebook

### PR DESCRIPTION
**Pull request by jmemich**
This is meant to showcase `ModelPipeline` functionality for users.

### CODE REVIEW

File: CivisML Example.ipynb [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+    "from civis.ml import ModelPipeline\n",
+    "from civis import APIClient\n",
+    "client = APIClient()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the first example, we'll use a random forest model from [scikit-learn](http://scikit-learn.org/stable/modules/ensemble.html#forest). In addtion, we'll grid search over hyperparameters for the maximum depth of the tree and the number of trees to optimally predict which users choose to upgrade to a premium service from the free version. Our data are in an AWS Redshift database on Civis Platform."
```
Comment (keithing) [inline CivisML Example.ipynb]: addtion -> addition

**Comment by jmemich** [comment]
@keithing I could have sworn I made an `examples` folder but, alas, senility comes too soon to the unlucky.

In addition, I checked with @alexm1309 and confirmed the tables/schema available to new platform users. The dynamic database name generation should work for free trail users.

File: examples/CivisML Example.ipynb [hunk 2]
```diff
@@ -0,0 +1,763 @@
+{
```
Comment (keithing) [inline examples/CivisML Example.ipynb]: Can we name this `CivisML.ipynb` since we're already in `/examples`?

**Comment by jmemich** [comment]
@keithing merged #80 into this PR.

*hopefully* tests pass.

**Pull request merged**